<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserBook</class>
 <widget class="QMainWindow" name="UserBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Книга</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Меню</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>981</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vlTable">
     <item>
      <layout class="QHBoxLayout" name="hlSearch">
       <item>
        <widget class="QLineEdit" name="leSearch">
         <property name="placeholderText">
          <string>Введите название</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSearch">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="twMyBooks">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbBookName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Название книги</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbBookDescription">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание книги</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbReviewsTitle">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отзывы</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>981</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="hlSwap">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QLabel" name="lbMyBook">
       <property name="text">
        <string>Моя книга</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leMyBook">
       <property name="maxLength">
        <number>13</number>
       </property>
       <property name="placeholderText">
        <string>Книга</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOfferSwap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Предложить обмен</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lbBookOwner">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Владелец: [username]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbBookISBN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ISBN: [isbn]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDescriptionTitle">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Описание</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twReviews">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>671</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbOffer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Предложить обмен</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
